contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,133 @@
+---
+title: How to pinpoint the culprits behind your slow web pages
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow web apps is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and a leading cause of customer churn.
+
+In this tutorial, we'll show you:
+  * The three most important metrics for measuring page load speed
+ 
+  *  How to identify opportunities to improve them using PostHog's [performance monitoring in session replays tool](https://posthog.com/blog/posthog-changelog#performance-monitoring-in-session-recordings).
+  * A cheat sheet to help you prioritize the most important areas to focus on.
+
+## Setting up performance monitoring
+
+First, ensure that session replays are [enabled]((https://posthog.com/docs/session-replay/manual)). Next, navigate to the Recordings tab on your PostHog Dashboard. Finally, click on the Network tab in the sidebar next to or below the video.
+
+![A screenshot showing where to find the performance monitoring in the session replays tool](https://i.imgur.com/nr0QSSy.png)
```

Comment:
Nit: Would be great to have a screenshot without the alerts at the top regarding hitting limits, inviting users etc...